configs/convnet__data-nonlin0.02.yaml

```diff
@@ -0,0 +1,40 @@
+experiment_name: 'ConvNet_nonlinearity0.02'
+checkpoint_monitor: 'loss'
+checkpoint_monitor_mode: 'min'
```

Comment:
Ideally we'd need to monitor some other metric than just training loss (because of overfitting). This applied to all the config files